crates/matrix-sdk-crypto/src/backups/mod.rs

```diff
@@ -619,7 +619,15 @@ impl BackupMachine {
             }
         }
 
-        self.store.import_room_keys(decrypted_room_keys, true, progress_listener).await
+        // This method is a bit flawed: we have no real idea which backup version these
```

Comment:
Could you annotate this with a `TODO: ` or `FIXME: `, so we can more easily find this fact.

Reply:
I can, but this is the main reason I deprecated the method. The real fix is to remove it.